index.html

```diff
@@ -1611,175 +1649,31 @@ <h3>The <dfn data-dfn-for="">Observable</dfn> interface</h3>
-        The <dfn>scheme</dfn> property represents the identification of the security scheme and MUST be <code>"apikey"</code>.
-      </p>
-    </section>
-
-    <section data-dfn-for="OAuth2SecurityScheme" >
-      <h3> The <dfn data-dfn-for="">OAuth2SecurityScheme</dfn> dictionary</h3>
-      <p>
-        Represents the OAuth2 authentication security configuration.
-      </p>
-      <pre class="idl">
-        dictionary OAuth2SecurityScheme: SecurityScheme {
-          DOMString scheme = "oauth2";
-          USVString tokenURL;
-          USVString authorizationURL;
-          USVString refreshURL;
-          sequence&lt;DOMString&gt; scopes;
-          DOMString flow = "implicit";
-        };
-      </pre>
-      <p>
-        The <dfn>scheme</dfn> property represents the identification of the security scheme and MUST be <code>"oauth2"</code>.
-      </p>
-      <p>
-        The <dfn>tokenURL</dfn> property represents the URL of the token server.
-      </p>
-      <p>
-        The <dfn>authorizationURL</dfn> property represents the URL of the authorization server.
-      </p>
-      <p>
-        The <dfn>refreshURL</dfn> property represents the URL of the refresh server.
-      </p>
-      <p>
-        The <dfn>scopes</dfn> property represents the authorization scope identifiers as an array of strings.
-      </p>
-      <p>
-        The <dfn>flow</dfn> property represents the authorization flow of type. Accepted values are: <code>"implicit"</code> (the default value), <code>"password"</code>, <code>"client"</code>, or <code>"code"</code>.
-      </p>
-      <p>
-         For the <code>"implicit"</code> flow the <code>authorizationURL</code> and <code>scopes</code> properties are required. For the <code>"password"</code> and <code>"client"</code> flows the <code>tokenURL</code> and <code>scopes</code> properties are required. For the <code>"code"</code> flow the <code>authorizationURL</code>, <code>tokenURL</code> and <code>scopes</code> properties are required.
-      </p>
-    </section>
+    <p>
+      In general, implementations SHOULD consider the following guidelines:
+      <ol>
+        <li>
+          Perform isolation of script instances (where applicable).
+        </li>
+        <li>
+          Perform security validation on all script inputs, including <a href="https://en.wikipedia.org/wiki/Fuzzing">fuzzing</a>.
+        </li>
+      </ol>
+    </p>
+    <p>
+      Script developers SHOULD consider the following guidelines:
+      <ol>
+        <li>
+          Minimize scripts functionality and complexity.
+        </li>
+        <li>
+          Avoid heavy functional processing in scripts without authentication.
+        </li>
+      </ol>
+    </p>
+    <p>
+      Security metadata is represented as dictionary objects of type <dfn>SecurityScheme</dfn> whose properties and sub-classes are defined in the <a href="https://w3c.github.io/wot-thing-description/#sec-security-vocabulary-definition">Thing Description specification</a>.
```

Comment:
This is quite lost here now. I recommend to move it to "Data types used in a Thing" like the other structures.